src/main/scala/eventstore/ProjectionsClient.scala

```diff
@@ -22,7 +22,7 @@ import scala.util.Try
  * See : https://github.com/EventStore/EventStore/blob/release-v3.9.0/src/EventStore.ClientAPI/Projections/ProjectionsClient.cs
  */
 protected[this] trait ProjectionsUrls {
-  def createProjectionUrl(name: String, mode: ProjectionMode = Continuous, allowEmit: Boolean = true): String = {
+  protected def createProjectionUrl(name: String, mode: ProjectionMode = Continuous, allowEmit: Boolean = true): String = {
```

Comment:
If i let them public, i expose them as part of the ProjectionsClient, i'm not sure we want to do that, the user of the ProjectionsClient should not have the choice to use it.

If you think we should expose them, we could let it as is but I feel like this clutters the api